auto_install/install.sh

```diff
@@ -296,6 +306,36 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update; ${PKG_MANAGER} upgrade --prune"
```

Comment:
Same as in uninstall script:
```suggestion
		UPDATE_PKG_CACHE="${PKG_MANAGER} update"
```